### DISCUSSION TITLE
[help] layout issue introduced w/ tar_watch_ui() in 1.7.1

### DISCUSSION THREAD

**Discussion by kwbyron-lilly** [ropensci/targets#1302] [Help] [answered]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

Using R 4.3.2, shiny 1.8.0
Upgrading targets to 1.7.1 causes a layout issue with tar_watch_ui()

Reverting my renv to 1.7.0 resolves the issue, I am thinking it's tied to the bslib dependency. That said, setting the bslib to previous versions didn't resolve the issue. 

targets 1.7.1
![image](https://github.com/user-attachments/assets/755e9130-2f99-4292-b078-2c4d9aa4cb5f)

targets 1.7.0
![image](https://github.com/user-attachments/assets/473fd46b-3317-4dd6-8f3b-5e5e21d8c374)
  
# Example, but you'll need to use renv to set the versions of targets
    library(shiny)
    library(targets)
    shinyApp(ui =  fluidPage(targets::tar_watch_ui("targets-shiny",  seconds = 60,  targets_only = TRUE)), 
                server = function(input, output) {targets::tar_watch_server("targets-shiny")})

**Comment by wlandau** [comment, marked as answer]
Thanks, looks like I needed to wrap the UI in `bslib::page()`. Fixed in 04c41db6d1f5e428619d05197231862acb235fce.
